scrapers/scrape_vd_tests.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import re
+import scrape_common as sc
+import scrape_vd_common as svc
+
+
+pdf_urls = svc.get_all_weekly_pdf_urls()
+for pdf_url in pdf_urls:
+    pdf = sc.pdfdownload(pdf_url, silent=True, page=1)
+    pdf = re.sub(r'(\d+)\'(\d+)', r'\1\2', pdf)
+    pdf = re.sub(r'(\d+)’(\d+)', r'\1\2', pdf)
+
+    td = sc.TestData(canton='VD', url=pdf_url)
+
+    year = sc.find(r'Situation au \d+.*(20\d{2})', pdf)
+    res = re.search(r'Entre\s+le\s+(\d+\s+\w+)\s+et\s+le\s+(\d+\s+\w+),', pdf)
+    if res:
+        td.start_date = sc.date_from_text(f'{res[1]} {year}').isoformat()
+        td.end_date = sc.date_from_text(f'{res[2]} {year}').isoformat()
+    else:
+        res = re.search(r'Entre\s+le\s+(\d+)\s+et\s+le\s+(\d+\s+\w+),', pdf)
+        if res:
+            end_date = sc.date_from_text(f'{res[2]} {year}')
+            td.end_date = end_date.isoformat()
+            td.start_date = sc.date_from_text(f'{res[1]}.{end_date.month}.{year}').isoformat()
+    assert td.start_date and td.end_date, f'failed to extract start and end dates from {pdf_url}'
+
+    res = re.search(r'une\s+moyenne\s+de\s+(\d+)\s+frottis\s+SARS-CoV(-)?2', pdf)
+    assert res, f'failed to extract total number of tests from {pdf_url}'
+    td.total_tests = 7 * int(res[1])
```

Comment:
As you mentioned, we only have a weekly average, so I think this is okay. But maybe it's better to calculate the difference between the `start_date` and `end_date` to be safe (instead of hard-coding `7`). WDYT?

Reply:
Sounds very reasonable! I've updated the PR (and also fixed the order for getting the PDFs, it should pick the latest ones available).